README.md

```diff
@@ -102,10 +102,11 @@ Usage in Node.js is the same, however you'll need to also require [node-canvas](
 
 ```js
 const mergeImages = require('merge-images');
-const Canvas = require('canvas');
+const { Canvas, Image } = require('canvas');
 
 mergeImages(['./body.png', './eyes.png', './mouth.png'], {
-  Canvas: Canvas
+  Canvas: Canvas,
+  Image: Image
```

Comment:
since this limits the usability to _only_ canvas v2 (since it's a dev, not full dependency, and the consumer would have to provide it, i think it'd be better to document how canvas v2 could be used on the consumer's end instead of forcing it through this route.

Reply:
I'm not quite following. `merge-images` would still work with canvas v1. In the v1 case the `Image` property would just be omitted.

Reply:
Ah, you're right, I was misreading this as the source, not the markdown. I was thinking something similar but just doing doc for it. Sorry for my misread

Reply:
ah cool :)